wp-content/themes/csisnuclearnetwork/assets/_scss/layout/_header.scss

```diff
@@ -327,31 +342,54 @@
   &__menu {
```

Comment:
I would maybe recommend moving the navigation styles into their own file and keeping the `_header.scss` file more layout + logo stuff, but up to you.

Don't forget to add a cursor to the buttons/links in the menu - the browser won't add that automatically to buttons, so you'll need to do it.